daemon/api_test.go

```diff
@@ -1764,6 +1766,50 @@ func (s *apiSuite) sideloadCheck(c *check.C, content string, head map[string]str
 	return chg.Summary()
 }
 
+func (s *apiSuite) runGetConf(c *check.C, keys []string) map[string]interface{} {
+	s.vars = map[string]string{"name": "test-snap"}
+	req, err := http.NewRequest("GET", "/v2/snaps/test-snap/conf?keys="+strings.Join(keys, ","), nil)
+	c.Check(err, check.IsNil)
+	rec := httptest.NewRecorder()
+	snapConfCmd.GET(snapConfCmd, req, nil).ServeHTTP(rec, req)
+
+	var body map[string]interface{}
+	err = json.Unmarshal(rec.Body.Bytes(), &body)
+	c.Check(err, check.IsNil)
+	return body["result"].(map[string]interface{})
+}
+
+func (s *apiSuite) TestGetConfSingleKey(c *check.C) {
+	d := s.daemon(c)
+
+	// Set a config that we'll get in a moment
+	d.overlord.State().Lock()
+	transaction, err := configstate.NewTransaction(d.overlord.State())
+	c.Check(err, check.IsNil)
+	transaction.Set("test-snap", "test-key", "test-value")
+	transaction.Commit()
+	d.overlord.State().Unlock()
+
+	result := s.runGetConf(c, []string{"test-key"})
+	c.Check(result, check.DeepEquals, map[string]interface{}{"test-key": "test-value"})
+}
+
+func (s *apiSuite) TestGetConfSingleMultipleKeys(c *check.C) {
+	d := s.daemon(c)
+
+	// Set a config that we'll get in a moment
+	d.overlord.State().Lock()
+	transaction, err := configstate.NewTransaction(d.overlord.State())
+	c.Check(err, check.IsNil)
+	transaction.Set("test-snap", "test-key1", "test-value1")
+	transaction.Set("test-snap", "test-key2", "test-value2")
+	transaction.Commit()
+	d.overlord.State().Unlock()
+
+	result := s.runGetConf(c, []string{"test-key1", "test-key2"})
```

Comment:
Instead of duplicating these two cases, can we have a single test, running runGetConf both with one entry and with two entries?

Reply:
What is this? 
# * Copyright (c) 1999 TaBE Project? My honest is hacked and every phone I get suffers the same. How do I stop this? I think it's a parent controlling device or in my wifi network. Is that possible to do remotely?